apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -621,16 +624,36 @@ class TrygdetidServiceImpl(
         }
     }
 
-    override fun overstyrNorskPoengaaarForTrygdetid(
+    override suspend fun overstyrNorskPoengaaarForTrygdetid(
         trygdetidId: UUID,
         behandlingId: UUID,
         overstyrtNorskPoengaar: Int?,
+        brukerTokenInfo: BrukerTokenInfo,
     ): Trygdetid {
         val trygdetid =
             trygdetidRepository.hentTrygdetidMedId(behandlingId, trygdetidId)
                 ?: throw GenerellIkkeFunnetException()
 
-        return trygdetidRepository.oppdaterTrygdetid(trygdetid.copy(overstyrtNorskPoengaar = overstyrtNorskPoengaar))
+        val oppdatertTrygdetid = trygdetidRepository.oppdaterTrygdetid(trygdetid.copy(overstyrtNorskPoengaar = overstyrtNorskPoengaar))
+
+        val datoer = hentDatoerForBehandlingOgAvdoed(behandlingId, oppdatertTrygdetid.ident, brukerTokenInfo)
+
+        val nyBeregnetTrygdetid =
+            beregnTrygdetidService.beregnTrygdetid(
+                trygdetidGrunnlag = oppdatertTrygdetid.trygdetidGrunnlag,
+                datoer.foedselsDato,
+                datoer.doedsDato,
+                oppdatertTrygdetid.overstyrtNorskPoengaar,
+            )
+
+        return when (nyBeregnetTrygdetid) {
+            null -> oppdatertTrygdetid.nullstillBeregnetTrygdetid()
+            else -> oppdatertTrygdetid.oppdaterBeregnetTrygdetid(nyBeregnetTrygdetid)
+        }.also { nyTrygdetid ->
+            trygdetidRepository.oppdaterTrygdetid(nyTrygdetid).also {
+                behandlingKlient.settBehandlingStatusTrygdetidOppdatert(behandlingId, brukerTokenInfo)
+            }
+        }
```

Comment:
```suggestion
        return when (nyBeregnetTrygdetid) {
            null -> oppdatertTrygdetid.nullstillBeregnetTrygdetid()
            else -> oppdatertTrygdetid.oppdaterBeregnetTrygdetid(nyBeregnetTrygdetid)
        }.also { nyTrygdetid ->
            trygdetidRepository.oppdaterTrygdetid(nyTrygdetid)
            behandlingKlient.settBehandlingStatusTrygdetidOppdatert(behandlingId, brukerTokenInfo)
        }
```

Reply:
Vil gjør det ukonsistent med andre steder med lignende kode - men - ser at vi kan kanskje hente ut en funksjon

Reply:
jeg tenker also nøstet i en also der vi ikke bruker parameteret i den indre blokken er ganske mye støy for å gjøre det konsekvent :p